src/csfloat/content_script.ts

```diff
@@ -634,6 +634,7 @@ async function adjustItem(container: Element, isPopout = false) {
             await caseHardenedDetection(container, cachedItem, false);
         }
         await addFadePercentages(container, cachedItem);
+        await addFloatColoring(container, cachedItem);
```

Comment:
I would like to see this being an option/setting, so users who dislike excessive coloring can turn it off. Should be on per default though. I'm fine with having it always-on in popups.

Reply:
> A global option for all websites and editing the skinport part or only for csfloat ?

Skinport has a setting 'Low/High Float Coloring' and CSFloat should have something similar. There are no real 'global' settings for now in the extension popup, but that may be a good point for the future!

Reply:
I made a big refactor to implement the site by site settings like for the colors to allow easy implementation on other sites

Reply:
Just committed a small change as your code had a small flaw regarding settings that do not follow the 'site-setting' naming scheme. Otherwise LGTM, will merge ASAP.

Reply:
thanks !